hub/management/commands/import_foe_groups.py

```diff
@@ -18,6 +18,7 @@ class Command(BaseImportFromDataFrameCommand):
     defaults = {
         "data_type": "integer",
         "category": "movement",
+        "subcategory": "groups",
```

Comment:
might be nice to add a data migration to automatically update these.

Reply:
I was thinking of just manually changing it in django admin as it isn't a huge amount of changes, is this a bad idea?

Reply:
No, that would also work :)